OptimizelySDK.Tests/UtilsTests/ExperimentUtilsTest.cs

```diff
@@ -40,17 +47,24 @@ public void TestIsUserInExperimentReturnsTrueWithNoAudience()
             experiment.AudienceIds = new string[] { };
             experiment.AudienceConditions = null;
 
-            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, null));
+            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, null, Logger));
+
+            LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, @"Evaluating audiences for experiment ""feat_with_var_test"": []"), Times.Once);
         }
 
         [Test]
         public void TestIsUserInExperimentReturnsTrueWhenAudienceUsedInExperimentNoAttributesProvided()
         {
             var experiment = Config.GetExperimentFromKey("feat_with_var_test");
             experiment.AudienceIds = new string[] { "3468206648" };
-
-            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, null));
-            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, new UserAttributes { }));
+            
+            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, null, Logger));
+            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, new UserAttributes { }, Logger));
+
+            LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, @"Evaluating audiences for experiment ""feat_with_var_test"": [""3468206648""]"), Times.Exactly(2));
+            LoggerMock.Verify(l => l.Log(LogLevel.DEBUG, @"Starting to evaluate audience ""3468206648"" with conditions: [""not"",{""name"":""input_value"",""type"":""custom_attribute"",""match"":""exists""}]"), Times.Exactly(2));
+            LoggerMock.Verify(l => l.Log(LogLevel.INFO, $@"Audience ""3468206648"" evaluated to TRUE"), Times.Exactly(2));
+            LoggerMock.Verify(l => l.Log(LogLevel.INFO, @"Audiences for experiment ""feat_with_var_test"" collectively evaluated to TRUE"), Times.Exactly(2));
```

Comment:
Why `Times.Exactly(2)` instead of `Times.Once`?

Reply:
The IsUserInExperiment above has been called twice. Once with null attributes and then with empty object.

Reply:
Ah, gotcha, thanks. That works for me!